src/kayenta/edit/editMetricModal.tsx

```diff
@@ -81,6 +81,33 @@ function FilterTemplateSelector({ metricStore, template, templates, select }: IF
   );
 }
 
+function EffectSizeSummary({ effectSizes }: { effectSizes: ICanaryMetricEffectSizeConfig }) {
+  if (!effectSizes) {
```

Comment:
might add a check for `Object.keys(effectSizes) === 0`

Reply:
yeah, that's a good idea. I hope we never end up leaving empty `effectSize` objects in metric data, but best to handle it in case we do.